src/charm.py

```diff
@@ -430,5 +434,13 @@ def flatten_config(value: Any, prefix: str = "") -> Dict[str, Any]:
     return flat
 
 
+def is_fake_authz_config(ext_authz_info) -> bool:
+    """Check if the provided external authorization configuration has fake values."""
+    return (
+        ext_authz_info.ext_authz_port == "fake_port"
+        and ext_authz_info.ext_authz_service_name == "fake_host"
+    )
+
+
```

Comment:
Can we implement this, as well as a generator for "fake", in the library?  As-is, the istio-k8s and istio-ingress charms both have their own implementation of what fake is, which has to be in sync.  Instead, let's have a single definition of what fake is (in the lib) and they each import that.  That way you can test that they're using the same definition